deploy/plays/deps.yml

```diff
@@ -1,12 +1,96 @@
 - hosts: machines
   become: yes
   tasks:
+    - name: Create an out file
+      file: path=./logs/deps.txt state=touch
+
+    - name: Clear the file
+      shell : truncate -s 0 ./logs/deps.txt
+
     - raw: which python || apt-get update
+      register: t1
```

Comment:
Is there a way to merge this task with the next one and avoid the use of temporary variables? Please check the Ansible docs for a possibility.

In the proposed code, there are temporary variables. If we can work without using any temporary variables, all the better.